Vagrantfile

```diff
@@ -47,6 +47,13 @@ Vagrant.configure(2) do |config|
     vb.cpus = 2
   end
 
+  host_user = ENV.fetch("USER", "vagrant")
+  aws_profile = ENV.fetch("RF_AWS_PROFILE", "raster-foundry")
+  rf_aws_config_bucket = ENV.fetch("RF_AWS_CONFIG_BUCKET",
```

Comment:
Ah, since some of this is necessary for our deployment scripts, I'll rename it to be consistent with what we were already calling it